.github/workflows/release-drafter-go.yaml

```diff
@@ -77,6 +80,11 @@ jobs:
       - uses: actions/setup-go@v5.0.2
         with:
           go-version-file: "${{ inputs.project-path }}/go.mod"
+      - name: Configure access to internal and private GitHub repos
+        env:
+          GITHUB_OAUTH_TOKEN: ${{ secrets.GITHUB_OAUTH_TOKEN }}
```

Comment:
Doing this to make it available for the "if" as referenced here:
https://stackoverflow.com/questions/72925899/github-actions-detect-if-secret-exists